workspace/build_target.go

```diff
@@ -164,7 +210,22 @@ func (bt BuildTarget) Build(ctx context.Context, runtimeCtx *runtime.Runtime, ou
 		}
 	}
 
-	LoadBuildPacks(ctx, builder, buildpacks)
+	buildPackStartTime := time.Now()
+	buildPackTimes, err := LoadBuildPacks(ctx, builder, buildpacks)
+	buildPacksEndTime := time.Now()
+	buildPacksTotalTime := buildPacksEndTime.Sub(buildPackStartTime)
```

Comment:
nit: This can be shortened:

```suggestion
	buildPacksTotalTime := time.Since(buildPackStartTime)
```